nbextensions/usability/toc2/main.js

```diff
@@ -83,20 +83,6 @@ define(["require", "jquery", "base/js/namespace",  'services/config',
     return a;
   };
 
-
-  var make_link_originalid = function (h, num_lbl) {
```

Comment:
It is no longer used anywhere. It seems like it might have been a hack to prevent the statically-generated TOC from having two copies of the section number in the first place, when this is not necessary. New code prevents this by simply using the same links as the floating TOC.